authn/authn.go

```diff
@@ -43,55 +50,79 @@ func NewClient(config Config) (*Client, error) {
 // ID and should be used as a unique foreign key in your users data.
 //
 // If the JWT does not verify, the returned error will explain why. This is for debugging purposes.
-func (ac *Client) SubjectFrom(idToken string) (string, error) {
-	claims, err := ac.verifier.GetVerifiedClaims(idToken)
+// It is also possible to overwrite the default audience used to verify the JWT by passing
+// a jwt.Audience{} as the second parameter.
+func (ac *Client) SubjectFrom(idToken string, options ...interface{}) (string, error) {
+	var verifier JWTClaimsExtractor
+
+	if len(options) == 0 {
+		verifier = ac.verifier
```

Comment:
may as well make this the default value when declaring `verifier`

Reply:
Good point. Changed with the last commit